jscode.js

```diff
@@ -0,0 +1,22 @@
+var words = ['It', 'is', 'awfully', 'hard'];
+var times = [1000, 2000, 4000, 6000];
+
+function showAyah(words,times){
+
+	var htmlElement = document.querySelector(".ayahWords");
+	let wordCount = words.length;
+
+	for(var currentIndex = 0; currentIndex < wordCount; currentIndex++) {
+		
+		(function(index) {
+			let word = words[index];//это необходимо?

```

Comment:
What is necessary? You can use `words[index]` and `times[index]` directly in `setTimeout` function.